transports/pom.xml

```diff
@@ -37,4 +37,12 @@
         </plugins>
     </build>
 
-</project>
\ No newline at end of file
+    <dependencies>
```

Comment:
Why so many projects need this dependency? Should not be just a transitive dependency of the runner?